docs/query_tool.md

```diff
@@ -126,3 +126,10 @@ This number will be the same across rows corresponding to the same subject.
 Example:
 
 {{ read_table('./repos/neurobagel_examples/query-tool-results/participant-level-results.tsv') }}
+
+#### Participant-level results in aggregate mode
+When `NB_RETURN_AGG` environment variable for the neurobagel API is set to `true` (see related section of the documentation [here](https://neurobagel.org/infrastructure/#set-the-environment-variables)), the value for all columns except for `DatasetID` and `SessionPath` in the Participant-level results is set to `protected`.
```

Comment:
Since most folks using the query tool may not necessarily be the same people who deployed the tool stack itself (that is, sysadmins who know about our env variables), it may be more clear if you flipped the two parts of your sentence around and add a bit more context for a pure query user.

e.g.,
"If the values for all columns except for `DatasetID` and `SessionPath` in the participant-level results are set to `protected`, this indicates that for data privacy reasons, the graph being queried has been configured (via its corresponding API) to return only aggregate information about matches. This behaviour arises when `NB_RETURN_AGG` environment variable ... etc.

I would also maybe add in the parentheses that the `true` value is the default. 🙂

Reply:
Good call!